src/util/type.h

```diff
@@ -141,6 +141,18 @@ class type_with_subtypet:public typet
   #endif
 };
 
+const type_with_subtypet &to_type_with_subtype(const typet &type)
```

Comment:
Here and all other newly introduced functions here: missing `inline`

Reply:
Done